src/components/Player.vue

```diff
@@ -641,13 +666,53 @@ li.move:not(.from) .player .overlay svg.move {
   top: 5px;
   box-shadow: 0 0 5px black;
   text-overflow: ellipsis;
-  overflow: hidden;
```

Comment:
Can't remove this I'm afraid, it would break long player names:
Now
![image](https://user-images.githubusercontent.com/325521/109548640-2efaba00-7acd-11eb-86fb-62b35ca854ec.png)
Before
![image](https://user-images.githubusercontent.com/325521/109548692-433eb700-7acd-11eb-8746-de70f0f56f19.png)